quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -62,71 +53,53 @@ object VitaminQuantityPickers {
         subtractEnabled: Boolean = true,
         editTextEnabled: Boolean = true,
         isExpanded: Boolean = false,
+        helperText: String? = null,
         keyboardOptions: KeyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
         keyboardActions: KeyboardActions = KeyboardActions.Default,
         colors: QuantityColors = VitaminQuantitiesColors.normal(),
-        shape: CornerBasedShape = VitaminTheme.shapes.radius100,
-        textStyle: TextStyle = VitaminTheme.typography.text2,
+        sizes: QuantitySizes = VitaminQuantitiesSizes.primary(),
         ripple: RippleTheme = VitaminTheme.ripples.brand,
         onAddClicked: () -> Unit,
         onSubtractClicked: () -> Unit,
         onValueChange: (String) -> Unit
     ) {
         CompositionLocalProvider(LocalRippleTheme provides ripple) {
-            Row(
+            val contentDescription = stringResource(id = R.string.vtmn_quantity_picker_accessibility_message, value)
```

Comment:
You never use your key `vtmn_quantity_picker_accessibility_message_with_helper` ?